### PR TITLE
thors-sql 1.5.6 (new formula)

### PR DESCRIPTION
**Pull request by Loki-Astari**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source thors-sql`?
- [x] Does your build pass `brew audit --online --strict thors-sql` (after doing `brew install thors-sql`)?
- [x] Does your build run tests `brew test thors-sql` ?

-----

### CODE REVIEW

**Comment by Loki-Astari** [comment]
This looks wrong.
There should only be one commit

**Comment by Loki-Astari** [comment]
OK. Only 1 file changes. Which is correct. Re-Opening.

**Comment by ilovezfs** [comment]
what is the output of `git remote`?

File: Formula/thors-sql.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+  needs :cxx14
+
+  def install
+    ENV["COV"] = "gcov"
+    system "mysql.server", "start"
+    system "init/testdb"
+    system "./configure", "--disable-vera",
+                          "--prefix=#{prefix}"
+    system "make", "-j", "1"
```
Comment (SMillerDev) [inline Formula/thors-sql.rb]: You can use `ENV.deparallelize` here instead.

File: Formula/thors-sql.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      :revision => "2da4ab817a4f45f0d3f8a7bf7ad0fc455031f0f6"
+
+  depends_on "mysql" => :build
+  depends_on "openssl"
+
+  needs :cxx14
+
+  def install
+    ENV["COV"] = "gcov"
+    system "mysql.server", "start"
```
Comment (SMillerDev) [inline Formula/thors-sql.rb]: Is this needed to build? Reading https://github.com/Loki-Astari/ThorsSQL/blob/master/doc/building.md makes it seem like that's only for tests.

File: Formula/thors-sql.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
+class ThorsSql < Formula
+  desc "SQL library for C++14"
+  homepage "https://github.com/Loki-Astari/ThorsSQL"
+  url "https://github.com/Loki-Astari/ThorsSQL.git",
+      :tag => "1.5.6",
+      :revision => "2da4ab817a4f45f0d3f8a7bf7ad0fc455031f0f6"
+
+  depends_on "mysql" => :build
```
Comment (SMillerDev) [inline Formula/thors-sql.rb]: This is also used in the tests

File: Formula/thors-sql.rb [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  depends_on "openssl"
```
Comment (SMillerDev) [inline Formula/thors-sql.rb]: Is there any configuration that can be used to specify which openSSL to use? Otherwise it might pick up Apples SSL.

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/thors-sql.rb (hunk 2 above)
Comment (Loki-Astari) [reply Formula/thors-sql.rb]: Its run as part of the unit tests.
Which happens during a build.

I could disable this.

File: Formula/thors-sql.rb (hunk 3 above)
Comment (Loki-Astari) [reply Formula/thors-sql.rb]: build and test

File: Formula/thors-sql.rb (hunk 1 above)
Comment (Loki-Astari) [reply Formula/thors-sql.rb]: Will update to do that.

File: Formula/thors-sql.rb (hunk 4 above)
Comment (Loki-Astari) [reply Formula/thors-sql.rb]: It links against: -lcrypto
It includes:        <openssl/sha.h>

**Comment by Loki-Astari** [comment]
@ilovezfs 

````
> git remote  -v
origin	git@github.com:Loki-Astari/homebrew-core.git (fetch)
origin	git@github.com:Loki-Astari/homebrew-core.git (push)
upstream	https://github.com/Homebrew/homebrew-core.git (fetch)
upstream	https://github.com/Homebrew/homebrew-core.git (push)
````

**Comment by ilovezfs** [comment]
That said, we'll need to pass on this formula for now as it's a self-submission, and the repository doesn't meet the notability requirements:
```
iMac-TMP:homebrew-core joe$ brew audit --new-formula thors-sql
thors-sql:
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
Error: 1 problem in 1 formula
```

@Loki-Astari you may want to host it in [your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap), though.

**Comment by Loki-Astari** [comment]
@ilovezfs No Problem.

It is already in my own Tap Loki-Astari/homebrew-ThorsAnvil     (I tested it there first).
I'll be back when its famous enough :-)

PS. When I run the `brew audit thors-sql --online --new-formula` I don't get that error message.

**Pull request closed without merging**